ui/app/components/actions/context.hbs

```diff
@@ -0,0 +1,13 @@
+<span class="hint-link connect-link" {{on "click" this.toggleHint}}>
+  {{inline-svg "bolt" class="nav-icon"}}
+  Connect
```

Comment:
A small nit from me here is I'm wondering if "Connect" implies it'll like somehow connect _to_ the UI. Feels a bit awkward language-wise. Could just "CLI" work? "Setup CLI" is probably the clearest but not sure if that really fits up there.

Reply:
Thought about this a bit - I reckon just "CLI" should work. I'll make the change and merge.